pkg/image/apis/image/v1/helpers.go

```diff
@@ -0,0 +1,229 @@
+package v1
```

Comment:
Could you vigorously prompt any potential editor to make the same changes to `pkg/image/apis/image/helper.go`?